hppc-benchmarks/build.gradle

```diff
@@ -18,6 +18,10 @@ jmh {
   duplicateClassesStrategy = DuplicatesStrategy.WARN
 }
 
+jmhJar {
+  duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Comment:
I added this strategy to be able to run benchmarks.

Reply:
What is actually duplicated? Duplicates in a ZIP file are highly suspicious - this means the unpacker typically picks one of the alternatives (with the same path). This can easily turn into a debugging nightmare. I haven't had the time to check but I can look into this - perhaps as part of this/ another issue?

Reply:
I just reran without this strategy to remember. I get this error:
> Task :hppc-benchmarks:jmhJar FAILED
> Entry LICENSE is a duplicate but no duplicate handling strategy has been set.